knative-operator/pkg/controller/knativekafka/rbacproxytransforms.go

```diff
@@ -0,0 +1,50 @@
+package knativekafka
+
+import (
+	"context"
+	"errors"
+
+	mf "github.com/manifestival/manifestival"
+	operatorv1alpha1 "github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis/operator/v1alpha1"
+	"github.com/openshift-knative/serverless-operator/openshift-knative-operator/pkg/monitoring"
+	"k8s.io/apimachinery/pkg/util/sets"
+	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	kafkaChannelComponents = []string{"kafka-ch-controller", "kafka-ch-dispatcher", "kafka-webhook"}
+	kafkaSourceComponents  = []string{"kafka-controller-manager"}
+)
+
+func addRBACProxySupportToManifest(instance *operatorv1alpha1.KnativeKafka, components []string) (*mf.Manifest, error) {
+	proxyManifest := mf.Manifest{}
+	// Only create the roles needed for the deployment service accounts as Prometheus has already
+	// the rights needed due to eventing that is assumed to be installed.
+	for _, c := range components {
+		crbM, err := monitoring.CreateClusterRoleBindingManifest(c, instance.GetNamespace())
+		if err != nil {
+			return nil, err
+		}
+		proxyManifest = proxyManifest.Append(*crbM)
+		if err = monitoring.AppendManifestsForComponent(c, instance.GetNamespace(), &proxyManifest); err != nil {
+			return nil, err
+		}
+	}
+	return &proxyManifest, nil
+}
+
+func getRBACProxyInjectTransformer(apiClient client.Client) (mf.Transformer, error) {
+	eventingList := &eventingv1alpha1.KnativeEventingList{}
```

Comment:
`KnativeEventingList` ?

Reply:
when requesting a namespaced resource the name may vary, so I didnt want to assume that the name is `knative-eventing`  because the user may change that no?